packages/core/solidity/src/print.ts

```diff
@@ -54,6 +56,20 @@ export function printContract(contract: Contract, opts?: Options): string {
   );
 }
 
+function printCompatibleLibraryVersions(contract: Contract): string[] {
+  const lines: string[] = [];
+  lines.push(`// Compatible with OpenZeppelin Contracts ${compatibleContractsSemver}`);
+  if (importsCommunityContracts(contract)) {
+    try {
+      const commit = getCommunityContractsGitCommit();
+      lines.push(`// Compatible with OpenZeppelin Community Contracts commit ${commit}`);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+  return lines;
```

Comment:
Let's try to use only 1 row, reading the same twice feels off

```suggestion
  const lines: string[] = [];
  let compat = `// Compatible with OpenZeppelin Contracts ${compatibleContractsSemver}`;
  if (importsCommunityContracts(contract)) {
    try {
      const commit = getCommunityContractsGitCommit();
      compat += `and Community Contracts commit ${commit}`
      lines.push(`// Compatible with OpenZeppelin Community Contracts commit ${commit}`);
    } catch (e) {
      console.error(e);
    }
  }
  lines.push(compat);
  return lines;
```